sickbeard/webserve.py

```diff
@@ -1619,12 +1619,26 @@ def deleteShow(self, show=None, full=0):
                 logger.log("Trakt: Unable to delete show: {0}. Error: {1}".format(showObj.name, ex(e)),logger.ERROR)
                 return self._genericMessage("Error", "Unable to delete show: {0}".format(showObj.name))
 
-        showObj.deleteShow(bool(full))
-
-        ui.notifications.message('%s has been %s %s' %
+        try:
+            from sickbeard.show_queue import ShowQueue
+            ShowActions = ShowQueue()
+            if not ShowActions._isInQueue(showObj,(ShowQueueActions.UPDATE, ShowQueueActions.FORCEUPDATE, ShowQueueActions.REFRESH, ShowQueueActions.RENAME, ShowQueueActions.SUBTITLE)) and not ShowActions._isBeingSomethinged(showObj, (ShowQueueActions.ADD, ShowQueueActions.UPDATE, ShowQueueActions.FORCEUPDATE, ShowQueueActions.REFRESH, ShowQueueActions.RENAME, ShowQueueActions.SUBTITLE) ):
```

Comment:
if not ShowActions._isInQueue(showObj,queue_actions) and not ShowActions._isBeingSomethinged(showObj, queue_actions):